common/persistence/client/factory.go

```diff
@@ -322,18 +325,11 @@ func (f *factoryImpl) init(clusterName string, limiters map[string]quotas.Limite
 	f.datastores[storeTypeVisibility] = visibilityDataStore
 }
 
-func buildRatelimiters(cfg *config.Persistence) map[string]quotas.Limiter {
+func buildRatelimiters(cfg *config.Persistence, persistenceMaxQPS dynamicconfig.IntPropertyFn) map[string]quotas.Limiter {
```

Comment:
nit: here since you are in the persistence package you can just refer to this as MaxQPS in this context.